raft/src/test/java/org/apache/kafka/raft/KafkaRaftClientTest.java

```diff
@@ -302,7 +303,7 @@ public void testGrantVotesWhenShuttingDown(boolean withKip853Rpc) throws Excepti
 
     @ParameterizedTest
     @ValueSource(booleans = { true, false })
-    public void testInitializeAsResignedAndBecomeCandidate(boolean withKip853Rpc) throws Exception {
```

Comment:
The diff is misleading here. This test was just removed because I found it was a duplicate of `testInitializeAsResignedLeaderFromStateStore`